### ISSUE TITLE
Error when trying to deploy the static plugins on Windows (from source or distro)

### ISSUE THREAD

**Issue by jcastro-dotcms** [dotCMS/core#8414]
Config | Value
------------ | -------------
dotCMS | 3.3, 3.5, 3.6
Failing env | Source code ONLY
OS | Windows 7
DB | Any
Java | 1.7

There's a problem when running the **deploy-war-tomcat** to build both the dotCMS source and the example "hello.world" static plugin on Windows. This happens **ONLY when compiling the plugin from the Git checkout**. Running the "deploy-plugins" script in the distro is working correctly. Also, on Mac OS X, both the source and the distro work correctly as well.

When you run the **gradlew deploy-war-tomcat", you get the following error:

```
[ant:echo] Deploying plugins to: C:\dotcms_releases\core/dotCMS | C:\dotcms_rele
ases\core/dotCMS/WEB-INF/lib
[ant:echo] Building at: com.dotcms.config at C:\dotcms_releases\core\plugins\com
.dotcms.config
[ant:javac] C:\dotcms_releases\core\plugins\common.xml:62: warning: 'includeantr
untime' was not set, defaulting to build.sysclasspath=last; set to false for rep
eatable builds
[ant:echo] Building at: training at C:\dotcms_releases\core\plugins\training
[ant:javac] C:\dotcms_releases\core\plugins\common.xml:62: warning: 'includeantr
untime' was not set, defaulting to build.sysclasspath=last; set to false for rep
eatable builds
[ant:javac] Note: C:\dotcms_releases\core\plugins\training\src\com\dotmarketing\
plugins\hello\world\HelloWorldContentletPreHook.java uses or overrides a depreca
ted API.
[ant:javac] Note: Recompile with -Xlint:deprecation for details.
:deploy FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':deploy'.
> The following error occurred while executing this line:
  C:\dotcms_releases\core\build.xml:570: The following error occurred while exec
uting this line:
  C:\dotcms_releases\core\build.xml:543: The following error occurred while exec
uting this line:
  C:\dotcms_releases\core\plugins\common.xml:112: Failed to create missing paren
t directory for C:\dotcms_releases\core\C:\dotcms_releases\core\dotCMS\WEB-INF\l
ib\plugin-training-classpath.jar

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug
option to get more log output.

BUILD FAILED

Total time: 17.312 secs
```

In this path in the stack trace: **C:\dotcms_releases\core\C:\dotcms_releases\core\dotCMS\WEB-INF\lib\plugin-training-classpath.jar**, notice how it's trying to append "\core\C:\dotc", the "C:" again in the path.

Temporary fix for the *common.xml* file: https://gist.github.com/oarrietadotcms/533015d33bc87c64ed39

**Comment by joseorsini** [comment]
Related to plugin deployment on Windows environment: We have noticed that a plugin that 3rd party jars cannot successfully compile in 3.3/3.3.1 running on Windows, because jars included in the plugin are never copied to WEB-INF/lib folder.

We have tried using the ROOT folder and recreate the path to {ROOT_CONTEXT}/WEB-INF/lib folder, and also we have tried with the plugin's lib folder, and the plugin still does not compile.

**Comment by jtesser** [comment]
this is a known issue and for clients they should use the distro and remote debug. or something like that

**Comment by jtesser** [comment]
This is of course a temporary workaround and we will address in the future

**Comment by joseorsini** [comment]
@jtesser the issue I reported happens on a Distro release.

**Comment by oarrietadotcms** [comment]
This issue bites us every time @jcastro-dotcms or me do a Java Dev Training and the client is using Windows. Gives the wrong impression. Could we increase the priority on this one, please?

**Comment by DeanGonzalez** [comment]
We just got another client complaint about this bug during Java training, can we get this in the next release?

**Comment by jgambarios** [comment]
Also please remove this line: https://github.com/dotCMS/core/blob/master-3.6/build.xml#L1001

It is affecting the deploy of static plugins from the source code.

**Comment by oarrietadotcms** [comment]
Could we, please, have this merge also in 3.5? The training will use master-3.5 until dotCMS 3.6 release.

**Comment by dsilvam** [comment]
@oarrietadotcms merged to master-3.5.

**Comment by maurizo** [comment]
Tested on Windows 7 on Source and Dist
Java 1.8
